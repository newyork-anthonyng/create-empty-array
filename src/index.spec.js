const createEmptyArray = require("./");

it("should create empty iterable array", () => {
  const result = createEmptyArray(5);
  const map = result.map(() => "Hi");

  expect(result).toMatchSnapshot();
  expect(map).toMatchSnapshot();
});

it("should not explode when length is not specified", () => {
  const result = createEmptyArray();

  expect(result).toMatchSnapshot();
});
