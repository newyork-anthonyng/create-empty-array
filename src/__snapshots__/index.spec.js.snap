// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create empty iterable array 1`] = `
Array [
  undefined,
  undefined,
  undefined,
  undefined,
  undefined,
]
`;

exports[`should create empty iterable array 2`] = `
Array [
  "Hi",
  "Hi",
  "Hi",
  "Hi",
  "Hi",
]
`;

exports[`should not explode when length is not specified 1`] = `
Array [
  undefined,
]
`;
